### DISCUSSION TITLE
Line limit in the generated code

### DISCUSSION THREAD

**Discussion by RocwoDev** [community/community#13510] [Copilot Conversations]
Hello,
I am very impressed by this extension which manages to generate working code in most cases and helps me a lot since I use it.

But I noticed that the code can't exceed a certain number of lines and if it does, the code I see is truncated (only the beginning of the code is displayed).

Is this a limitation that will be exceeded in the future or is it a setting problem on my side? (I found only few parameters)

I use visual studio code.

Thanks for your help

**Comment by MileneNojoda** [comment]
Same issue here, how can I increase the suggestion limit?

**Comment by gibovskii** [comment]
I suspect it's a limit of ChatGPT 3. The same occurs in the web based ui.
